source/_components/sensor.octoprint.markdown

```diff
@@ -35,9 +35,21 @@ sensor:
       - Time Remaining
 ```
 
+{% configuration %}
+name:
+  description: Name to use in the frontend.
+  required: true
+  type: string
+  default: Octoprint
+monitored_conditions:
+  description: Conditions to monitor.
+  required: true
+  type: list
+{% endconfiguration %}
+
 Configuration variables:
 
-- **name** (*Optional*): The name of the sensor. Default is 'OctoPrint'.
+- **name** (*Required*): The name of the sensor. This needs to match a name under the Octoprint platform configuration.
```

Comment:
This is now duplicate? Since you've already created a new list just above this, using the new formatting.